src/ensembl/production/datacheck/static/css/ensembl.css

```diff
@@ -0,0 +1,215 @@
+h2 {
+  padding-top: 20px;
+  padding-bottom: 10px;
+}
+
+a {
+  color: #337ab7;
+}
+
+fieldset {
+  margin-bottom: 10px;
+}
+
+legend {
+  margin-bottom: 0;
+}
+
+.select-align-right {
+  text-align-last: right;
+}
+
+select.select-align-right > option {
+  direction: rtl;
+}
+
+.form-group {
+  margin-bottom: 0.75rem;
+}
+
+.navbar {
+  padding: 0rem 0rem 0rem 1rem;
+  margin-bottom: 20px;
+}
+
+.navbar-brand {
+  margin-right: 0.5rem;
+}
+
+.navbar-expand-lg .navbar-nav .nav-link {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.nav-link.active {
+  background-color: rgba(128, 0, 102, 0.2);
+}
+
+.table td, .table th {
+  padding: .5rem 0.75rem;
+}
+
+.col-form-label {
+  text-align: right;
+}
+
+.fg-purple {
```

Comment:
This purple is a bit aggressive for my taste, but it never seemed worth the effort to change it; now might be a good time to adjust it though, so something a bit softer?